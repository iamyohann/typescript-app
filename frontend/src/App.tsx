import * as React from 'react';
import './App.css';

class App extends React.Component<any, any> {
  public render() {
    return (
      <div className="App" />
    );
  }
}

export default App;
