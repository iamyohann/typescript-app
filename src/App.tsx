import * as React from 'react';
import './App.css';

import TodoBar from './containers/TodoBar';

class App extends React.Component<any, any> {
  public render() {
    return (
      <div className="App">
        <TodoBar />
      </div>
    );
  }
}

export default App;
