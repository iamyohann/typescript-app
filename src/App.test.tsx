declare var window: Window;


import * as React from 'react';
import * as ReactDOM from 'react-dom';
import * as renderer from 'react-test-renderer';
import App from './App';

it('renders without crashing', () => {
    const div = window.document.createElement('div');
    ReactDOM.render(<App />, div);
    ReactDOM.unmountComponentAtNode(div);
});

it('renders according to the snapshot', () => {
  const tree = renderer
    .create(<App />)
    .toJSON();
  expect(tree).toMatchSnapshot();
  expect(1).toEqual(1);
});