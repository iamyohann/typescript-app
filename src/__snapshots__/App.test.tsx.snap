// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders according to the snapshot 1`] = `
<div
  className="App"
>
  <input
    placeholder="Type something..."
  />
  <button>
    Add
  </button>
</div>
`;
